enhancements/network/ovn-hypershift-live-migration.md

```diff
@@ -0,0 +1,295 @@
+be used, ignoring them is they are not the latest one.
+
+### Risks and Mitigations
+
+### OVN extended arp_proxy features ####
+
+
+The current ovn arp_proxy is missing the following features: enforcing the mac 
+address and ipv6, ideally those get implemented and ovn-kubernetes have 
+them for 4.14 release.
+
+In case they don't arrive on time, it will not be able to configure IPv6 hosted
+clusters at hypershift and VMs arp table will need to be refreshed after live
+migration. 
+
+### Open Questions 
+
+Do this impact non hypershift payloads ? The control-plane will have some more
+load from the /32 routing but data-plane is not affected
+
+Do this work with future interconnect topology ? It's not using a distributed 
+switch per hosted cluster so that part should be fine, but it may needed 
+to replicate /32 routing at all node's ovn_cluster_router.
+
+Do this need extra configuration at openshift virtualization ? The bridge 
+binding pod network live migration feature gate need to be activated
+
+### Test Plan
+
+Add some network tests to (hypershift conformance)[https://github.com/openshift/origin/pull/27456]
+that will do live migration with the following scenarios:
+- TCP connection between VMs
+- TCP connection to VMs from service
+- TCP connection from VM to service
+
+The connection has to survive and downtown be minimal.
+
+
+### Graduation Criteria
+
+At hypershift kubevirt provider GA this feature will be graduated since it unblocks the 
+live migration feature
+
+#### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+- Enumerate service level indicators (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
+
+#### Tech Preview -> GA
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+- Document SLOs for the component
+- Conduct load testing
+- User facing documentation created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+#### Removing a deprecated feature
+n/a
+### Upgrade / Downgrade Strategy
+
+Since hypershift is tech preview before this enhancement Upgrade/Downgrade is 
+not supported for hosted clusters, but it should not break openshift 
+networking for other payloads on upgrade or downgrade.
+
+### Version Skew Strategy
+n/a
+
+### Operational Aspects of API Extensions
+
+#### Failure Modes
+
+Probable failure situations include
+- OVN control plane too loaded by /32 routing
+  - Too much CPU / Memory
+  - Too slow a transaction rate
+  - Disk is almost full
+
+
+#### Support Procedures
+
+The CNV network team will be responsible for debugging hypershift kubevirt live 
+migration issues, alerts related to those routed should be helpful. 
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation
+History`.
```

Comment:
could you link the section instead ?

Reply:
Well, this is a leftover from the template, I will add the content.